### ISSUE TITLE
/table/ paths with 500 errors - PR tables do not exist or iterations do not exist

### ISSUE THREAD

**Issue by tuchandra** [censusreporter/censusreporter#185]
Creating this issue to keep track of topic pages where the PR tables do not exist or the iterations do not exist, but the topic page says they do. Related to #174, #175, #176.

No PR tables:
```
Migration - [u'B07404', u'B07004']
Race and Hispanic Origin - [u'B07404', u'B06004', u'B07004']
```

No iterations:
```
Poverty - [u'B21007', u'B21006', u'B27017', u'B27018', u'B18035', u'B18030', u'B18031', u'B18032', u'B18033', u'B18131', u'B18130', u'B05010', u'B08522', u'B18034', u'B18036', u'B14006', u'B08122', u'B27016', u'B16009']
```

I'll take care of these shortly.

**Comment by tuchandra** [comment]
The three distinct tables that do not have PR tables are actually cases of 173 -- there are PR iterations, but no core PR table. The code currently doesn't handle this, so I'll look into fixing that separately

**Comment by tuchandra** [comment]
Fixed by #186
